api/src/main/java/org/eclipse/microprofile/config/spi/Converter.java

```diff
@@ -69,12 +69,7 @@
  * the {@code Config} provides an <em>Implicit Converter</em>, if</p>
  * <ul>
  *     <li>The target type {@code T} has a Constructor with a String parameter, or</li>
- *     <li>The target type {@code T} has a Constructor with a CharSequence parameter, or</li>
  *     <li>the target type {@code T} has a {@code static T valueOf(String)} method, or</li>
- *     <li>the target type {@code T} has a {@code static T valueOf(CharSequence)} method, or</li>
- *     <li>the target type {@code T} has a {@code static T parse(String)} method, or</li>
- *     <li>the target type {@code T} has a {@code static T parse(CharSequence)} method, or</li>
- *     <li>the target type {@code T} is an {@code Enum}, in which case the {@link Enum#valueOf(Class, String)} is being used.</li>
  * </ul>
```

Comment:
The line about Enum should be kept

Reply:
We discussed this and John suggested to remove it as it is covered by the valueOf. @johnament commented: Technically, this last case isn't needed.  When an Enum type is created, it automatically gets a T.valueOf method.
It is implementation detail. The enum is covered under valueOf.

Reply:
ok, that's fine.